master/reference/kube-controllers/configuration.md

```diff
@@ -1,17 +1,24 @@
 ---
-title: Configuring the Calico policy controller
+title: Configuring the Calico Kubernetes controllers
 ---
 
-The policy controller is primarily configured through environment variables. When running
-the policy controller as a Kubernetes pod, this is accomplished through the pod manifest `env`
+The Kubernetes controllers are primarily configured through environment variables. When running
```

Comment:
I don't think you can refer to them as "The Kubernetes controllers". Its a bit ambiguous. I think Calico's Kubernetes Controllers makes more sense but it also personifies the software which isn't good.

Could stick with "The Calico Kubernetes Controllers are primarily..."

Reply:
Though I'll also add that referring to them as just "the controllers" later seems fine.

Reply:
There's a few places throughout that just refer to them as Kubernete Controllers so make sure to get em all

Reply:
Good points, @ozdanborne we could call them Calico Kubernetes controllers (lowercase "controllers") although it is a little bit like "Pepsi Coke soda" if you know what I mean. I don't have a strong opinion but might prefer just Kubernetes controller (IMO)